From 010b2f3b99d3f979968c8a37cdde734d5a314550 Mon Sep 17 00:00:00 2001
From: Daniel Kaminski de Souza <daniel@kryptonunite.com>
Date: Mon, 22 Feb 2021 18:32:24 +0000
Subject: [PATCH] Add even more transparency

---
 themes/Default Dark.css | 23 +++++++++++++++++++++--
 1 file changed, 21 insertions(+), 2 deletions(-)

diff --git a/themes/Default Dark.css b/themes/Default Dark.css
index c58a6dc..eb35379 100644
--- a/themes/Default Dark.css	
+++ b/themes/Default Dark.css	
@@ -2,8 +2,12 @@
   box-shadow: none !important;
 }
 
-.minimap, .editor-scrollable>.decorationsOverviewRuler {
-  opacity: 0.6;
+.minimap {
+  opacity: 0.5;
+}
+
+.editor-scrollable>.decorationsOverviewRuler {
+  opacity: 0;
 }
 
 .editor-container {
@@ -21,7 +25,22 @@
 .monaco-workbench,
 .monaco-workbench .part,
 body {
+  --raw_black: 0, 0, 0;
+  --raw_white: 255, 255, 255;
+  --primary_raw_color: var(--raw_white);
+  --secondary_raw_color: var(--raw_black);
+  /* --primary_raw_color: var(--raw_black);
+  --secondary_raw_color: var(--raw_white); */
+  --shadow_opacity: 0.9;
+  --shadow: 2.5px rgb(var(--secondary_raw_color), var(--shadow_opacity));
   background: transparent !important;
+  -webkit-font-smoothing: antialiased;
+  -webkit-text-stroke: 0.12px rgb(var(--primary_raw_color));
+  text-shadow:
+    -1px -1px var(--shadow),
+    1px 1px var(--shadow),
+    -1px 1px var(--shadow),
+    1px -1px var(--shadow)
 }
 
 .editor-group-container>.tabs {
-- 
2.20.1

